pinot-core/src/test/java/org/apache/pinot/queries/RangeQueriesTest.java

```diff
@@ -118,16 +122,28 @@ public void setUp()
     driver.init(segmentGeneratorConfig, new GenericRowRecordReader(records));
     driver.build();
 
-    IndexLoadingConfig indexLoadingConfig = new IndexLoadingConfig();
-    indexLoadingConfig.setRangeIndexColumns(
-        new HashSet<>(Arrays.asList(DICTIONARIZED_INT_COL, RAW_INT_COL, RAW_LONG_COL, RAW_FLOAT_COL, RAW_DOUBLE_COL)));
+    _noDictionaryColumns = new HashSet<>(Arrays.asList(RAW_INT_COL, RAW_LONG_COL, RAW_FLOAT_COL, RAW_DOUBLE_COL));
```

Comment:
Similar here, since we are not testing reloading, we should just create one table config, and use it to create and load the segment